02_configure_host.sh

```diff
@@ -153,6 +153,13 @@ ansible-playbook \
     -i ${VM_SETUP_PATH}/inventory.ini \
     -b -vvv ${VM_SETUP_PATH}/setup-playbook.yml
 
+# NOTE(elfosardo): /usr/share/OVMF/OVMF_CODE.fd does not exist in the ovmf
+# package anymore, so we need to create a link to that until metal3-dev-env
+# fixes that, probably when switching to UEFI by default
+if ! [[ -f /usr/share/OVMF/OVMF_CODE.fd ]]; then
+  ln -s /usr/share/edk2/ovmf/OVMF_CODE.fd /usr/share/OVMF/
```

Comment:
Doesn't it need sudo?

Reply:
that's a good question, I tested it and it worked without sudo, I didn't realize it was missing until I actually thought about it
CI looks good, but we're running it as root, so it doesn't count
I will add it to be on the safe side